src/libraries/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -952,13 +952,14 @@ private static unsafe int Read(SafeFileHandle fd, Span<byte> buffer)
             }
         }
 
-        internal static void WriteToTerminal(ReadOnlySpan<byte> buffer, bool mayChangeCursorPosition = true)
+        internal static void WriteToTerminal(ReadOnlySpan<byte> buffer, SafeFileHandle? overrideHandle = null, bool mayChangeCursorPosition = true)
```

Comment:
nit: I would not call the argument "**override**Handle". Since it's nullable I would just call it handle: when I provide a value, it's used, when I don't then some default value is used.